tests/templates/test_integration.py

```diff
@@ -176,558 +138,331 @@
+               {'features': [1., 2.], 'weights': qml.init.qaoa_embedding_uniform(n_layers=3, n_wires=2),
+                'wires': range(2)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1., 2.], 'weights': qml.init.qaoa_embedding_uniform(n_layers=3, n_wires=2),
+                'wires': range(2)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1., 2., 3.], 'weights': qml.init.qaoa_embedding_normal(n_layers=2, n_wires=3),
+                'wires': range(3)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1., 2., 3.], 'weights': qml.init.qaoa_embedding_normal(n_layers=2, n_wires=3),
+                'wires': range(3)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1.], 'weights': qml.init.qaoa_embedding_normal(n_layers=2, n_wires=1),
+                'wires': range(1)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1.], 'weights': qml.init.qaoa_embedding_uniform(n_layers=2, n_wires=1),
+                'wires': range(1)}),
+              (qml.templates.SimplifiedTwoDesign,
+               {'initial_layer_weights': qml.init.simplified_two_design_initial_layer_uniform(n_wires=4),
+                'weights': qml.init.simplified_two_design_weights_uniform(n_layers=3, n_wires=4),
+                'wires': range(4)}),
+              (qml.templates.SimplifiedTwoDesign,
+               {'initial_layer_weights': qml.init.simplified_two_design_initial_layer_normal(n_wires=4),
+                'weights': qml.init.simplified_two_design_weights_normal(n_layers=3, n_wires=4),
+                'wires': range(4)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_uniform(n_layers=1, n_wires=1), 'wires': range(1)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_uniform(n_layers=3, n_wires=1), 'wires': range(1)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_uniform(n_layers=3, n_wires=2), 'wires': range(2)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_uniform(n_layers=3, n_wires=3), 'wires': range(3)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_normal(n_layers=3, n_wires=1), 'wires': range(1)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_normal(n_layers=3, n_wires=2), 'wires': range(2)}),
+              (qml.templates.BasicEntanglerLayers,
+               {'weights': qml.init.basic_entangler_layers_normal(n_layers=3, n_wires=3), 'wires': range(3)}),
+              ]
+
+CV_INIT = [(qml.templates.CVNeuralNetLayers,
+            {'theta_1': qml.init.cvqnn_layers_theta_uniform(n_layers=3, n_wires=2),
```

Comment:
FYI: each entry appears twice in here as well

Reply:
Thanks! I wanted the duplicates to test the 1-wire edge case!